### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by cskksc**
This doesn't relate to any bug/feature. Noticed this while working on some other stuff.

- [ ] The commits are consistent with our contribution guidelines.
- [ ] You've added tests (if possible) to cover your change(s)
- [ ] All tests are passing (`make test`)
- [ ] The new code is not generating bytecode or `M-x checkdoc` warnings
- [ ] You've updated the changelog (if adding/changing user-visible functionality)
- [ ] You've updated the readme (if adding/changing user-visible functionality)
- [ ] You've updated the refcard (if you made changes to the commands listed there)

### CODE REVIEW

File: cider-client.el [hunk 1]
```diff
@@ -1101,10 +1101,10 @@ If CONN is not provided the user will be prompted to select a connection."
   "Change the DESIGNATION in cider buffer names.
 Buffer names changed are cider-repl and nrepl-server."
   (interactive (list (read-string (format "Change CIDER buffer designation from '%s': "
-					  (cider-extract-designation-from-current-repl-buffer)))))
+                                          (cider-extract-designation-from-current-repl-buffer)))))
   (cider-ensure-connected)
   (let ((new-repl-buffer-name (nrepl-format-buffer-name-template
-                                nrepl-repl-buffer-name-template designation)))
```
Comment (bbatsov) [inline cider-client.el]: Are you sure about this?

Comment (cskksc) [inline cider-client.el]: Yep. It's a function, and this indentation matches with the style guide too.

Comment (Malabarba) [inline cider-client.el]: Looks right to me

Comment (bbatsov) [inline cider-client.el]: Haven't been coding for a while. Must be getting crazy and rusty. :-)

Comment (cskksc) [inline cider-client.el]: :-)

**Pull request merged**